app/controllers/application_controller.rb

```diff
@@ -16,7 +16,7 @@ class ApplicationController < ActionController::Base
     # This secret is reset to a value found in the miq_databases table in
     # MiqWebServerWorkerMixin.configure_secret_token for rails server, UI, and
     # web service worker processes.
-    protect_from_forgery :secret => SecureRandom.hex(64), :except => :csp_report, :with => :exception
+    protect_from_forgery :secret => SecureRandom.hex(64), :except => [:authenticate, :csp_report], :with => :exception
```

Comment:
/cc @jvlcek if this is related to the ext-auth issue you were troubleshooting, we might need to add exceptions for the other two methods in dashboard_controller, namely :external_authenticate (for ext-auth), and :kerberos_authenticate (SSO).   Thanks.